.github/workflows/doc_build.yml

```diff
@@ -1,21 +1,93 @@
-name: Doc building
+name: Docs
 
 on: [push, pull_request]
 
 jobs:
-
-  doc_building:
+  plots:
+    name: Plots
     runs-on: ubuntu-latest
+    env:
+      PROJ_EXE: proj
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-
-      - name: Install Requirements
+      - uses: goanpeca/setup-miniconda@v1
+        with:
+          channels: conda-forge
+          auto-update-conda: true
+      - name: Setup
+        shell: bash -l {0}
         run: |
-          sudo apt update
-          sudo apt install -y python3-pip python3-setuptools python3-dev python3-wheel doxygen gcc
-          sudo pip3 install sphinx sphinx-rtd-theme sphinxcontrib-bibtex breathe
-
-      - name: Build doc
+            conda update -n base -c defaults conda
+            conda install descartes shapely geopandas matplotlib proj
+      - name: Plot
+        shell: bash -l {0}
+        working-directory: ./docs/plot
         run: |
-            (cd docs; make html)
+            ./plot.sh
```

Comment:
plot.sh will run with a PROJ version which is not the current one. So if we add a new projection method, this will likely fail.

Reply:
> plot.sh will run with a PROJ version which is not the current one. 

Indeed. My plan was to get both this and the Conda builder active and then make the plots depend on and use the conda builder.